src/components/layout/organize/TabbedLayout.tsx

```diff
@@ -13,6 +13,9 @@ interface StyleProps {
 }
 
 const useStyles = makeStyles<Theme, StyleProps>((theme) => ({
+    main: {
+        overflowX: 'hidden',
+    },
```

Comment:
What is this for?

Reply:
Material UI slide transitions along the horizontal cause flashing unless a container high enough up in the DOM has overflow disabled. I have a feeling if you use MUI Grid all the way down then you don't have this problem, but that's a guess...